docs/root/development/performance/cpu_impact.rst

```diff
@@ -0,0 +1,59 @@
+.. _dev_performance_cpu:
+
+.. _ios_envoy_example_app: https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/swift/hello_world
```

Comment:
`:repo:` can be used for all of these

Reply:
https://github.com/lyft/envoy-mobile/pull/209#discussion_r298781525